lib/middleware/errors.js

```diff
@@ -22,7 +22,7 @@ function _applicationError (err, req, res, next) {
   next(err);
 }
 
-function _authenticationError (err, req, res, next) {
+function _authenticationError (err, req, res) {
```

Comment:
These are error middleware so I'm fairly sure they expect next as the 4th argument